test/snippet/range/view/interleave.cpp

```diff
@@ -0,0 +1,26 @@
+#include <range/v3/algorithm/equal.hpp>
+
+#include <seqan3/io/stream/debug_stream.hpp>
+#include <seqan3/range/view/interleave.hpp>
+
+using namespace seqan3;
+
+int main()
+{
+//![general]
+std::string u{"FOOBARBAXBAT"};
+std::string i{"in"};
+size_t s = 3;
+std::string cmp{"FOOinBARinBAXinBATin"};
+std::string cmp_rev{"niTAB"};
+
+// pipe notation
+// explicitly call view::all
+auto v0 = u | view::interleave(s, i);
+// or, via constructor.
+auto v1 = view::interleave(u, s, i);
+
+if (ranges::equal(v0, v1))
+    debug_stream << "Success!\n";
+//![general]
```

Comment:
This looks more like a test than a snippets, maybe instead just a little shorter:
```cpp
std::string u{"FOOBARBAXBAT"};
std::string i{"in"};
size_t s = 3;

auto v = u | view::interleave(s, i);

debug_stream << v << '\n'; // prints FOOinBARinBAXinBATin
```
(main stuff around it of course, too)

Reply:
I think the policy is now to include the full snippet files in the API doc with the includes and main function, not a snippet from it.

Reply:
Now that the full file is included, please indent the contents of main() like you would normally!

Reply:
it seems you forgot to include the final `}`. Note that you can get rid of these `//![general]` ans just `\include` instead of `\snippet` above to get the whole.